webapp/src/app/pages/components/knowledge-graph/concept-map/concept-map.component.ts

```
@@ -1984,13 +2051,32 @@
   }
 
   async previewSlide(event, slideId: Object) {
-    this.docURL = `${this.cmEndpointURL}${this.currentMaterial?.url}${this.currentMaterial?._id}.pdf`
-    this.currentPDFPage = parseInt(slideId["value"]);
+    this.docURL = `${this.cmEndpointURL}${this.currentMaterial?.url}${this.currentMaterial?._id}.pdf`;
+    this.currentPDFPage = parseInt(slideId['value']);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)